client/my-sites/plan-features-2023-grid/components/billing-timeframe.tsx

```diff
@@ -71,16 +72,27 @@ function usePerMonthDescription( {
 				: null;
 
 		if ( fullTermPriceText ) {
+			const isEnglishLocale = currentLocale === 'en' || currentLocale?.startsWith( 'en-' );
```

Comment:
Nit: We can also replace this with the `useIsEnglishLocale` hook from `@automattic/i18n-utils`.

There is also the `useHasEnTranslation` hook from the same package that returns a function that checks if the locale is English or if the translation is present. The naming of this hook is slightly confusing, so I'll leave it up to you if you want to use it. 🙂

Reply:
Good suggestion! I went with `useIsEnglishLocale` because I prefer the readability 👍

[eb695eb](https://github.com/Automattic/wp-calypso/pull/76738/commits/eb695ebb0ea7f1fc8adb00dc7dc14b67db54230f)